UPGRADE.md

```diff
@@ -17,6 +17,25 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 * [shopsys/product-feed-zbozi]
 
 ## [From 7.0.0-alpha2 to Unreleased]
+### [shopsys/coding-standards]
+- In order to run all checks, there is new unified way - execute `php vendor/bin/ecs check /path/to/project --config=vendor/shopsys/coding-standards/easy-coding-standard.yml`
+    - See [EasyCodingStandard docs](https://github.com/Symplify/EasyCodingStandard#usage) for more information
+##### Example of custom configuration file
+```yaml
+#custom-coding-standard.yml
+imports:
+    - { resource: '%vendor_dir%/symplify/easy-coding-standard/config/psr2.yml' }
+    - { resource: '%vendor_dir%/shopsys/coding-standards/shopsys-coding-standard.yml' }
+parameters:
+    exclude_files:
+        - '*/ignored_folder/*'
+    skip:
+        # private properties should not start with "_"
+        PHP_CodeSniffer\Standards\Squiz\Sniffs\NamingConventions\ValidVariableNameSniff.PrivateNoUnderscore: ~
+        # skip max length of functions in files
+        ObjectCalisthenics\Sniffs\Files\FunctionLengthSniff:
+            - '%current_working_dir%/src/file.php'

```

Comment:
This variable shortcut (`%current_working_dir%`) works only on `imports` section, see https://github.com/symplify/easyCodingStandard/#use-prepared-checker-sets

Reply:
changed to */src